From e60dca679cc4684287b97c00e47183d3d52055a5 Mon Sep 17 00:00:00 2001
From: Rajaganesh Rathinasabapathi <Rajaganesh.Rathinasabapathi@amd.com>
Date: Sun, 10 Jul 2022 22:54:27 -0500
Subject: [PATCH] increase firmware update max timeout

increase max timeout to 7 mins in firmware update page.
BIOS update programs two SPIs with one update image
which is around 6 mins.

Signed-off-by: Rajaganesh Rathinasabapathi <Rajaganesh.Rathinasabapathi@amd.com>
---
 src/views/Operations/Firmware/FirmwareFormUpdate.vue | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/views/Operations/Firmware/FirmwareFormUpdate.vue b/src/views/Operations/Firmware/FirmwareFormUpdate.vue
index 9c4149c..2c25861 100644
--- a/src/views/Operations/Firmware/FirmwareFormUpdate.vue
+++ b/src/views/Operations/Firmware/FirmwareFormUpdate.vue
@@ -171,7 +171,7 @@ export default {
         this.endLoader();
         this.errorToast(this.$t('pageFirmware.toast.updateAborted'));
       } else {
-        if (this.informProgress.overalltimeout > 210000) {
+        if (this.informProgress.overalltimeout > 420000) {
           this.informProgress.overalltimeout = 0;
           this.endLoader();
           this.infoToast(this.$t('pageFirmware.toast.verifyUpdateMessage'), {
